repotracker/repotracker.go

```diff
@@ -697,6 +698,15 @@ func shellVersionFromRevision(ref *model.ProjectRef, metadata model.VersionMetad
 			}
 			v.RevisionOrderNumber = num
 		}
+	} else if metadata.GitTag.Tag != "" {
+		if !utility.StringSliceContains(ref.GitTagAuthorizedUsers, metadata.GitTag.Pusher) {
```

Comment:
Should this just be silently ignored? I feel like it's fine that someone who isn't on the list pushes a tag to a repo but I could be missing something

Reply:
This shouldn't be called if you don't pass this authorization; I'm just adding this in again since shellVersionFromRevision is accessible from a rest route so I want to guarantee this authorization somehow (although that's still not perfect but)